docs/whats-new/staging-release-notes.md

```diff
@@ -1,5 +1,55 @@
 # Staging release notes
 
+## Staging release 03 August 2022
+Frontend version: x | Backend version: x  
+**New features/Updates** - **TechPass Portal**
```

Comment:
is this a new feature or improvement?

If it is New feature, we just mention "New features" and no need for /Updates

Reply:
1 new feature 1 new update.